AppLock/src/main/java/com/infomaniak/lib/applock/LockActivity.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Infomaniak Core - Android
+ * Copyright (C) 2023 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.lib.applock
+
+import android.os.Bundle
+import android.util.Log
+import androidx.activity.addCallback
+import androidx.appcompat.app.AppCompatActivity
+import com.infomaniak.lib.applock.databinding.ActivityLockBinding
+import com.infomaniak.lib.core.utils.Utils.FACE_ID_LOG_TAG
+import com.infomaniak.lib.core.utils.getAppName
+import com.infomaniak.lib.core.utils.requestCredentials
+
+class LockActivity : AppCompatActivity() {
+
+    private val binding by lazy { ActivityLockBinding.inflate(layoutInflater) }
+
+    override fun onCreate(savedInstanceState: Bundle?) = with(binding) {
+        super.onCreate(savedInstanceState)
+        setContentView(root)
+        if (savedInstanceState?.getBoolean("firstLaunch") != false) {
```

Comment:
```suggestion
        if (savedInstanceState?.getBoolean("firstLaunch") == true) {
```
Is it ok to remove this negation, or not ?

Reply:
This is old code from kDrive  which I extracted, so I don't know if it changes something.
I would consider leaving it as it was to avoid surprise regression